configs/sample_yamls/serviceops/sample_service_config.yml

```diff
@@ -0,0 +1,172 @@
+#
+########################## types ########################
+#
+types:
+# QUOTA
+- name: ratelimit_by_service_by_consumer
+  quota_params:
```

Comment:
I think there is value in grouping all these things under "types" and "constructors". Otherwise, it's not clear what you're looking at in config.